File: FeedleDataTier/FeedleDataTier/Network/UpdateUserRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class UpdateUserRequest : Request
    {
        [JsonPropertyName("user")]
        public User User { get; set; }

        public UpdateUserRequest(User user) : base(RequestType.UpdateUser)
        {
            this.User = user;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/UpdateUserRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.User;

public class UpdateUserRequest extends Request{
    private User user;
    public UpdateUserRequest(User user)
    {
        super(RequestType.UpdateUser);
        this.user = user;
    }

    public User getUser() {
        return user;
    }
}
